code/modules/research/nanites/nanite_programs/protocols.dm

```diff
@@ -46,43 +46,25 @@
 	factory_efficiency = min(factory_efficiency + 1, max_efficiency)
 	nanites.adjust_nanites(null, round(0.002 * factory_efficiency, 0.1))
 
-/datum/nanite_program/protocol/tinker
-	name = "Tinker Protocol"
-	desc = "Replication Protocol: the nanites learn to use metallic material in the host's bloodstream to speed up the replication process."
+/datum/nanite_program/protocol/pyramid
+	name = "Pyramid Protocol"
+	desc = "Replication Protocol: the nanites implement an alternate cooperative replication protocol that is more efficient as long as the saturation level is above 80%."
 	use_rate = 0
 	rogue_types = list(/datum/nanite_program/necrotic)
 	protocol_class = NANITE_PROTOCOL_REPLICATION
-	var/boost = 2
-	var/list/valid_reagents = list(
-		/datum/reagent/iron,
-		/datum/reagent/copper,
-		/datum/reagent/gold,
-		/datum/reagent/silver,
-		/datum/reagent/mercury,
-		/datum/reagent/aluminium,
-		/datum/reagent/silicon)
-
-/datum/nanite_program/protocol/tinker/check_conditions()
-	if(!nanites.host_mob.reagents)
-		return FALSE
-
-	var/found_reagent = FALSE
+	var/boost = 1.2
```

Comment:
This seems rather high. The highest you can get in nanite replication speed without harming the user that doesnt use protocols is 1.1, so this is more than double that speed.

Reply:
Without using protocols is the operative word here, they do require much more effort than the regular nanite tech tree programs. It is on a similar level to the other replication protocols; for comparison Kickstart gives 3.5 for the first two minutes from implantation, Factory scales up to 2 volume per second after about 15 minutes, and Eclipse gives an extra 3 while unconscious.

That said, it is the most reliable and easy to benefit from of the replication protocols, so a nerf might be warranted. After all I'm not an expert in balancing numbers and i usually don't get much feedback from nanite users.